pkg/dedup/iter.go

```diff
@@ -15,6 +16,17 @@ import (
 	"github.com/thanos-io/thanos/pkg/store/storepb"
 )
 
+var (
+	initialPenalty int64 = 5000
+	mutex          sync.Mutex
+)
+
+func InitialPenalty(value int64) {
```

Comment:
Can we inject the penalty into the dedup iterator instead of setting the global variable?